mains/gdas.cc

```diff
@@ -96,6 +100,7 @@ int main(int argc,  char ** argv) {
   // Check that the application is recognized
   // ----------------------------------------
   const std::set<std::string> validApps = {
+    "convertstructuredgrid",
```

Comment:
@jrocha22: I agree with @CoryMartin-NOAA's comment that this should probably be `converttostructuredgrid` for consistency.

Reply:
sounds good!

Reply:
Looks great, thanks!